fixture-monkey-api/src/test/java/com/navercorp/fixturemonkey/api/random/RegexGeneratorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.navercorp.fixturemonkey.api.random;
+
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.api.BDDAssertions.thenThrownBy;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.junit.jupiter.api.Test;
+
+public class RegexGeneratorTest {
```

Comment:
`public` is not required.

Reply:
it may be better to move to the `kotlin-tests` module.
It is more readable thanks to local class.

Reply:
I thought it would be better to have the test in the same module as the `RegexGenerator`. Do you prefer moving it?

Reply:
Yes, it is better to test as a client.

Tests in the same module may have a access to a package-private method or class that is not intended.

Reply:
You are right, in this case, tests in the same module would be better.